src/deephaven/plugin/__init__.py

```diff
@@ -12,14 +16,30 @@ class Plugin(abc.ABC):
 
 
 class Registration(abc.ABC):
+    """
+    Registration types should be set as the registration_cls for deephaven.plugin entrypoints for their package to
+    ensure that they are all run on server startup.
+    """
     class Callback(abc.ABC):
```

Comment:
In this case, we really should have a space, since the docstring has is not for the inner class.